srcpkgs/python-pycryptodome/template

```diff
@@ -9,10 +9,10 @@ hostmakedepends="python-setuptools python3-setuptools"
 makedepends="python-devel python3-devel"
 short_desc="Python2 package of low-level cryptographic primitives"
 maintainer="Wietse Kuipers <wietse@kuiprs.nl>"
-license="Public Domain, BSD-2-Clause"
```

Comment:
The license file states 

```
The source code in PyCryptodome is partially in the public domain and partially released under the BSD 2-Clause license.
```

so the license= ist correct, please leave it as is.

Reply:
On second though, it might be better to change the first part to `custom:Public Domain` sice Public Domain is not a SPDX identifier.

Reply:
Now we recognize _Public Domain_ as license, please change back.